spec/features/pipelines/pipelineRun_status_test.go

```diff
@@ -0,0 +1,48 @@
+package pipelines
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-pipelines/release-tests/pkg/config"
+	"github.com/openshift-pipelines/release-tests/pkg/helper"
+	"github.com/openshift-pipelines/release-tests/pkg/pipelines"
+	flags "github.com/openshift-pipelines/release-tests/spec/flags"
+)
+
+var _ = Describe("PipelineRun and TaskRun status", func() {
+
+	// Setup up state for each test spec
+	// create new project (not set as active) and new context directory for each test spec
+	// This is before every spec (It)
+	BeforeEach(func() {
+		SetDefaultEventuallyTimeout(config.Timeout)
+		SetDefaultConsistentlyDuration(config.ConsistentlyDuration)
+		flags.Clients, flags.Namespace, flags.Cleanup = helper.NewClientSet()
+	})
+
+	AfterEach(func() {
+		defer flags.Cleanup()
+	})
+
+	Describe("Operator installed", func() {
+		When("I create a namespace", func() {
+			When("I create Task and pipeline ", func() {
+				It("I should Run Task and pipeline with `inexistence` Service account", func() {
```

Comment:
1. unify the SA name in all places
2. I guess that `non-existent` is grammatically correct